pdf/model/optimize/optimizer.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package optimize
+
+import (
+	. "github.com/unidoc/unidoc/pdf/core"
+	"github.com/unidoc/unidoc/pdf/model"
+)
+
+// New creates a optimizers chain from options
```

Comment:
Usually recommended to return the implementation (Chain).
Any disadvantage of that here?